packages/ai/mcp-stdio/index.ts

```diff
@@ -0,0 +1,4 @@
+export {
+  StdioClientTransport,
+  type McpStdioServerConfig,
```

Comment:
There is a mismatch here - for consistency we should either prefix both or none with `Mcp`

Also why `ClientTransport` vs `Transport`? could simplify, also if `Client` is used elsewhere. Same for `Server`:

```
export {
   StdioTransport,
   StdioConfig
}
```

Reply:
The naming here is explicit because:

- The transport we make available is specific to MCP clients!
- The config is essentially the server params you pass to the client transport!

I can see how the naming is a bit confusing but this aligns with official SDK naming conventions, happy to pivot away on our end!